packages/renderer-commonmark/src/index.ts

```diff
@@ -114,7 +135,7 @@ export default class CommonmarkRenderer {
    * Asterisks are used here because they can split
    * words; underscores cannot split words.
    */
-  *'bold'(node: HIRNode): IterableIterator<string> {
+  *'bold'(node: Annotation): Iterable<any> {
```

Comment:
Would it be more readable to also rename `node` to `annotation`?

Reply:
ahh, yeah. could we follow up on that?